apps/meteor/app/lib/server/startup/settings.ts

```diff
@@ -1555,8 +1555,11 @@ settingsRegistry.addGroup('Layout', function () {
 			type: 'boolean',
 			public: true,
 		});
-		this.add('Layout_Custom_Body', false, {
+		this.add('Layout_Custom_Body_Only', false, {
```

Comment:
is a new setting really needed? this is not *renaming* the old setting, it is creating a new one.

if a new setting is really required, the old one should be removed.

Reply:
Hi @sampaiodiego , yes the setting is really needed, the migration was not created because the 'old' setting was not released yet, it was only merged at develop branch couple of days ago.

Reply:
I see, thx @hugocostadev .. we'll need to be more careful on things like these once we start rolling upgrades even before they got released, but for now, I think we're good 👍